src/main/java/com/google/swarm/tokenization/DLPTextToBigQueryStreamingV2.java

```diff
@@ -227,14 +227,25 @@ private static void runInspectAndDeidPipeline(
                 .setJobName(options.getJobName())
                 .setDlpApiRetryCount(options.getDlpApiRetryCount())
                 .setInitialBackoff(options.getInitialBackoff())
-                .build())
-        .get(Util.inspectOrDeidSuccess)
-        .apply(
-            "StreamInsertToBQ",
-            BigQueryDynamicWriteTransform.newBuilder()
-                .setDatasetId(options.getDataset())
-                .setProjectId(options.getProject())
+                .setDataSinkType(options.getDataSinkType())
                 .build());
+
+    if(options.getDataSinkType() == Util.DataSinkType.GCS)
+      inspectDeidRecords.get(Util.deidSuccess).apply("WriteToGCS",
+              WriteToGCS.newBuilder()
+                      .setOutputBucket(options.getOutputBucket())
+                      .setFileType(options.getFileType())
+                      .build());
+    else if(options.getDataSinkType() == Util.DataSinkType.BigQuery)
```

Comment:
@dup05 Have we confirmed if we want this to be either BQ or GCS and not both or as per need basis?

Reply:
Yes @Goutam1511, Multiple sinks are not required